generator/internal/dart/annotate.go

```diff
@@ -571,9 +571,11 @@ func createToJsonLine(field *api.Field, state *api.APIState, required bool) stri
 	} else if isMap {
 		// primitive maps
 		return name
-	} else if isMessage || isEnum || isBytes {
+	} else if isMessage || isEnum {
 		// message, enum, and custom: name!.toJson()
 		return name + bang + ".toJson()"
+	} else if isBytes {
+		return "encodeBytes(" + name + bang + ")"
```

Comment:
This is more a comment about line 571...  `map<... , bytes>` is legal proto, and they do appear in Google Cloud APIs:

https://github.com/googleapis/googleapis/blob/2b63b9276dca4d5046470fe5e1022bd982629f62/google/cloud/confidentialcomputing/v1/service.proto#L291

I am not sure if we are handling this correctly, if not, fixing that is not urgent.

Reply:
I don't think we would handle it correctly; I'll open an issue to track it.